backend/models/Block.js

```diff
@@ -34,7 +34,7 @@ module.exports = (sequelize, DataTypes) => {
         allowNull: false
       },
       gasPrice: {
-        type: DataTypes.STRING,
+        type: DataTypes.BIGINT,
```

Comment:
We need to revert this change.

BIGINT in Sqlite is i64, but NEAR amounts are in u128, so we cannot store them in BIGINT (it would be more efficient to encode these values into varchar[8], but that requires quite some work, so it is easier to have STRING here). It is unfortunate as I would not expect gasPrice to go over i64 limit, but it is safer to match the API and consider fixing this on the nearcore side first.

Reply:
I am also really confused about the type of gas price from block. It is string type in the docs but when I write tests in nearlib, it shows it is number. So I do not know which is right

Reply:
@icerove Can you link to the code where it is defined as a number?